SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
```

Comment:
while this is a great idea, but sort of impractical because once EIP becomes final you wouldn't be able to update this section and the outdated info will be even more dangerous. its best if this for implementations is maintained somewhere else or an informational EIP be created to publish updates that resolve critical vulnerabilities